modules/cborm/models/BaseBuilder.cfc

```diff
@@ -557,8 +557,8 @@ component accessors="true"{
 		var partialSQL = "";
 		projection.sql = "";
 		// if multiple subqueries have been specified, smartly separate them out into a sql string that will work
-		for( var x=1; x<=listLen( arguments.rawProjection.sql ); x++ ) {
-			partialSQL = listGetAt( arguments.rawProjection.sql, x );
+		for( var x=1; x<=listLen( arguments.rawProjection.sql, "&&" ); x++ ) {
```

Comment:
I am not sure I follow why the adding of the `&&` can you refresh my memory why?